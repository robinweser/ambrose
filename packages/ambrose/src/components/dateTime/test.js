import React from 'react'

import DateTime from '.'
import Text from '../text'

import createSnapshot from '../../testing/createSnapshot'

describe('<DateTime />', () => {
  it('renders correctly with default props', () => {
    const snapshot = createSnapshot(
      <DateTime>{new Date(2021, 1, 15)}</DateTime>
    )
    expect(snapshot).toMatchSnapshot()
  })
  it('renders correctly when nested', () => {
    const snapshot = createSnapshot(
      <Text variant="heading">
        <DateTime>{new Date(2021, 1, 15)}</DateTime>
      </Text>
    )
    expect(snapshot).toMatchSnapshot()
  })
})
