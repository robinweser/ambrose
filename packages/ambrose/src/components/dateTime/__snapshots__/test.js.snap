// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DateTime /> renders correctly when nested 1`] = `
".cls_0 {
  display: contents;
}
.cls_1 {
  font-size: 20px;
  margin: 0;
  display: block;
  color: black;
}


<div className=test-theme cls_0>
  <div className=cls_1>15 februari 2021</div>
</div>;
"
`;

exports[`<DateTime /> renders correctly with default props 1`] = `
".cls_0 {
  display: contents;
}


<div className=test-theme cls_0>15 februari 2021</div>;
"
`;
